shared/offline/index.desktop.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import React from 'react'
+import {Box, Text, Icon} from '../common-adapters'
+import {globalStyles, globalColors} from '../styles'
+import {ReachabilityReachable} from '../constants/types/flow-types'
+import {ignoreDisconnectOverlay} from '../local-debug.desktop.js'
+
+import type {Props} from './index'
+
+const Offline = ({reachability}: Props) => {
+  if (reachability && reachability.reachable !== ReachabilityReachable.no) {
```

Comment:
reachability could be required (default to unknown) and can remove `reachability &&`